<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMenJinManagerClass</class>
 <widget class="QMainWindow" name="CMenJinManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMenJinManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,6,0,0">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="m_lbServerConfig">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>服务配置</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>203</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>  Ip     :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_editIp"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>后端端口 :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_editSPort"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>前端端口 :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_editCPort"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="m_btnOk">
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_btnQuit">
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CMenJinManager.qrc"/>
 </resources>
 <connections/>
</ui>
